modules/ossm-auto-route.adoc

```diff
@@ -44,6 +44,13 @@ gateway1-scqhv www.bookinfo.com            istio-ingressgateway   <all>
 
 If the gateway is deleted, {ProductName} deletes the routes. However, routes created manually are never modified by {ProductName}.
 
+[id="ossm-auto-route-annotations_{context}"]
+= Annotations
+
+Sometimes specific annotations are needed in an OpenShift Route. For example, some advanced features in OpenShift Routes are managed via xref:../../networking/routes/route-configuration.adoc[special annotations]. For this and other use cases, {ProductName} will copy all annotations present in the Istio Gateway resource (with the exception of those starting with `kubectl.kubernetes.io`) into the managed OpenShift Route resource.
+
+Thus, if you need specific annotations to be present in the OpenShift Routes created by {ProductName}, just add them to the Istio Gateway resource and they will be copied over.
```

Comment:
Suggestion for rewording, "If you need specific annotations in the OpenShift Routes created by {ProductName}, create then in the Istio Gateway resource and {ProductName} will copy them into the OpenShift Route resources managed by the {ProductName}  Operator.

Reply:
Done